src/ops/project.rs

```diff
@@ -89,10 +167,20 @@ impl Ingredient for Project {
                 for i in e {
                     new_r.push(r[*i].clone());
                 }
+                match self.expressions {
+                    Some(ref e) => {
+                        for i in e {
+                            new_r.push(self.eval_expression(i, r));
+                        }
+                    }
+                    None => (),
+                }
                 match self.additional {
-                    Some(ref a) => for i in a {
-                        new_r.push(i.clone());
-                    },
+                    Some(ref a) => {
+                        for i in a {
+                            new_r.push(i.clone());
+                        }
+                    }
```

Comment:
@malte this is a similar case to aggregations where columns appear in a pre-defined location (i.e., first all projected columns, then all expressions, then all literals). I know that affected MIR for aggregations during the deadline, so maybe be on the lookout for that here too?